src/index.html

```diff
@@ -9,9 +9,160 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <label class="form-element">Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-element">Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-element">How old are You?
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="age"
+          >
+        </label>
+
+        <label class="form-element">Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+
+        <label class="form-element">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="policyAgreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+
+        <label class="form-element">E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-element">Password:
+          <input
+            type="text"
+            name="password"
+            minlength="7"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-element">Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              value="yes"
+              name="loveToCats"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              value="no"
+              name="loveToCats"
+            >
+            No
+          </label>
+        </label>
```

Comment:
because it's entirely wrapped in label tag when I click anywhere on empty space in a row with "Yes" and "No" options it selects "Yes". Fix it please